### PR TITLE
Remove unused track ids

### PR DESCRIPTION
**Pull request by megaslug**
Removing tracking ids which we no longer need

### CODE REVIEW

File: app/pages/features/features.de.js [hunk 1]
```diff
@@ -48,37 +48,37 @@ export default class FeaturesDe extends React.Component {
...
-                  <a href='#pricing' id='track-sticky-nav-pricing' className='sticky-nav__link'>
+                  <a href='#pricing' className='sticky-nav__link'>
                     Preise
                   </a>
                 </li>
               </ul>
               <IfLocale hasInstantSignup>
                 <Href to='signup.path'
-                id='track-sticky-nav-merchants-new'
+
```
Comment (megaslug) [inline app/pages/features/features.de.js]: Remove extra line

File: app/pages/features/features.es.js [hunk 2]
```diff
@@ -50,37 +50,37 @@ export default class FeaturesEs extends React.Component {
...
-                  <a href='#pricing' id='track-sticky-nav-pricing' className='sticky-nav__link'>
+                  <a href='#pricing' className='sticky-nav__link'>
                     Tarifas
                   </a>
                 </li>
               </ul>
               <IfLocale hasInstantSignup>
                 <Href to='signup.path'
-                  id='track-sticky-nav-merchants-new'
+
```
Comment (megaslug) [inline app/pages/features/features.es.js]: Remove extra line

File: app/pages/features/features.nl.js [hunk 3]
```diff
@@ -42,37 +42,37 @@ export default class FeaturesNl extends React.Component {
...
-                  <a href='#pricing' id='track-sticky-nav-pricing' className='sticky-nav__link'>
+                  <a href='#pricing' className='sticky-nav__link'>
                     Tarieven
                   </a>
                 </li>
               </ul>
               <IfLocale hasInstantSignup>
                 <Href to='signup.path'
-                id='track-sticky-nav-merchants-new'
+
```
Comment (megaslug) [inline app/pages/features/features.nl.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.de.js [hunk 4]
```diff
@@ -43,23 +43,23 @@ export default class PartnerWithUsDe extends React.Component {
...
               <li className='sticky-nav__item'>
-                <a href='#examples' id='track-sticky-nav-features' className='sticky-nav__link'>
+                <a href='#examples' className='sticky-nav__link'>
                   Beispiele
                 </a>
               </li>
             </ul>
             <Link to='contact_sales' query={{ s: 'partners' }}
-            id='track-partners-sticky-nav-contact-sales'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.de.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.de.js [hunk 5]
```diff
@@ -150,7 +150,7 @@ export default class PartnerWithUsDe extends React.Component {
                 </p>
                 <Link to='contact_sales'
                 query={{ s: 'partners' }}
-                id='track-partners-find-out-more'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.de.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.de.js [hunk 6]
```diff
@@ -419,7 +419,7 @@ export default class PartnerWithUsDe extends React.Component {
               </p>
               <Link to='contact_sales'
               query={{ s: 'partners' }}
-              id='track-partners-contact-sales'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.de.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.en.js [hunk 7]
```diff
@@ -49,23 +49,23 @@ export default class PartnerWithUsEn extends React.Component {
...
               <li className='sticky-nav__item'>
-                <a href='#examples' id='track-sticky-nav-features' className='sticky-nav__link'>
+                <a href='#examples' className='sticky-nav__link'>
                   Examples
                 </a>
               </li>
             </ul>
             <Link to='contact_sales' query={{ s: 'partners' }}
-            id='track-partners-sticky-nav-contact-sales'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.en.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.en.js [hunk 8]
```diff
@@ -160,7 +160,7 @@ export default class PartnerWithUsEn extends React.Component {
                 </Translation>
                 <Link to='contact_sales'
                 query={{ s: 'partners' }}
-                id='track-partners-find-out-more'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.en.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.en.js [hunk 9]
```diff
@@ -423,7 +423,7 @@ export default class PartnerWithUsEn extends React.Component {
               </p>
               <Link to='contact_sales'
               query={{ s: 'partners' }}
-              id='track-partners-contact-sales'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.en.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.es.js [hunk 10]
```diff
@@ -418,7 +418,7 @@ export default class PartnerWithUsEs extends React.Component {
               </p>
               <Link to='contact_sales'
               query={{ s: 'partners' }}
-              id='track-partners-contact-sales'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.es.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.fr.js [hunk 11]
```diff
@@ -42,23 +42,23 @@ export default class PartnerWithUsFr extends React.Component {
...
               <li className='sticky-nav__item'>
-                <a href='#examples' id='track-sticky-nav-features' className='sticky-nav__link'>
+                <a href='#examples' className='sticky-nav__link'>
                   Exemples
                 </a>
               </li>
             </ul>
             <Link to='contact_sales' query={{ s: 'partners' }}
-            id='track-partners-sticky-nav-contact-sales'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.fr.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.fr.js [hunk 12]
```diff
@@ -148,7 +148,7 @@ export default class PartnerWithUsFr extends React.Component {
                 </p>
                 <Link to='contact_sales'
                 query={{ s: 'partners' }}
-                id='track-partners-find-out-more'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.fr.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.fr.js [hunk 13]
```diff
@@ -418,7 +418,7 @@ export default class PartnerWithUsFr extends React.Component {
               </p>
               <Link to='contact_sales'
               query={{ s: 'partners' }}
-              id='track-partners-contact-sales'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.fr.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.nl.js [hunk 14]
```diff
@@ -42,23 +42,23 @@ export default class PartnerWithUsNl extends React.Component {
...
               <li className='sticky-nav__item'>
-                <a href='#examples' id='track-sticky-nav-features' className='sticky-nav__link'>
+                <a href='#examples' className='sticky-nav__link'>
                   Voorbeelden
                 </a>
               </li>
             </ul>
             <Link to='contact_sales' query={{ s: 'partners' }}
-            id='track-partners-sticky-nav-contact-sales'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.nl.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.nl.js [hunk 15]
```diff
@@ -151,7 +151,7 @@ export default class PartnerWithUsNl extends React.Component {
                 </p>
                 <Link to='contact_sales'
                 query={{ s: 'partners' }}
-                id='track-partners-find-out-more'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.nl.js]: Remove extra line

File: app/pages/partner-with-us/partner-with-us.nl.js [hunk 16]
```diff
@@ -428,7 +428,7 @@ export default class PartnerWithUsNl extends React.Component {
               </p>
               <Link to='contact_sales'
               query={{ s: 'partners' }}
-              id='track-partners-contact-sales'
+
```
Comment (megaslug) [inline app/pages/partner-with-us/partner-with-us.nl.js]: Remove extra line

File: app/pages/pro/pro.de.js [hunk 17]
```diff
@@ -35,30 +35,30 @@ export default class ProDe extends React.Component {
...
                 <li className='sticky-nav__item'>
-                  <a href='#pricing' id='track-sticky-nav-pricing' className='sticky-nav__link'>
+                  <a href='#pricing' className='sticky-nav__link'>
                     Preis
                   </a>
                 </li>
               </ul>
               <Link to='contact_sales' query={{ s: 'pro' }}
-              id='track-sticky-nav-contact-sales'
+
```
Comment (megaslug) [inline app/pages/pro/pro.de.js]: Remove extra line

File: app/pages/pro/pro.en.js [hunk 18]
```diff
@@ -29,30 +29,30 @@ export default class ProEn extends React.Component {
...
                 <li className='sticky-nav__item'>
-                  <a href='#pricing' id='track-sticky-nav-pricing' className='sticky-nav__link'>
+                  <a href='#pricing' className='sticky-nav__link'>
                     Pricing
                   </a>
                 </li>
               </ul>
               <Link to='contact_sales' query={{ s: 'pro' }}
-              id='track-sticky-nav-contact-sales'
+
```
Comment (megaslug) [inline app/pages/pro/pro.en.js]: Remove extra line

File: app/pages/pro/pro.es.js [hunk 19]
```diff
@@ -36,30 +36,30 @@ export default class ProEs extends React.Component {
...
                 <li className='sticky-nav__item'>
-                  <a href='#pricing' id='track-sticky-nav-pricing' className='sticky-nav__link'>
+                  <a href='#pricing' className='sticky-nav__link'>
                     Tarifas
                   </a>
                 </li>
               </ul>
               <Link to='contact_sales' query={{ s: 'pro' }}
-              id='track-sticky-nav-contact-sales'
+
```
Comment (megaslug) [inline app/pages/pro/pro.es.js]: Remove extra line

**Comment by megaslug** [comment]
@jamesshedden could you give this a look?

**Comment by jamesshedden** [comment]
@megaslug Great work! Few more quick comments 😊

**Comment by jamesshedden** [comment]
cc @Ken-Lo quick heads up that we're removing the unused `id="track-..."` attributes as discussed :)

**Comment by megaslug** [comment]
@jamesshedden all changes have been made. Take another quick look whenever you can. Thanks.

**Comment by jamesshedden** [comment]
@megaslug Great work! Nice one 👍

**Comment by Ken-Lo** [comment]
Woo hoo! Great work @megaslug :-) These IDs have been annoying me for a while, great to finally be rid of them! :-)

**Pull request merged**